src/workloads/execution/PipelineUpdate.yml

```diff
@@ -47,193 +47,244 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2, 4, 6, 8]
+      NopInPhasesUpTo: 9
+      PhaseConfig:
+        Repeat: 1
 
 - Name: RunUpdates
   Type: RunCommand
   Threads: 16
   Phases:
   - *Nop
   - *Nop
-  - Repeat: 500    # Classic updates on small documents phase.
+  - *Nop
+  - Repeat: 300    # Classic updates on small documents phase.
+    GlobalRate: 2 per 100 milliseconds
     Database: *db
     Operations:
     - OperationMetricsName: ClassicUpdateAddFieldsSmallDoc
       OperationName: RunCommand
+      OperationIsQuiet: true
```

Comment:
Is there another ticket to make true the default or could this change be implemented here. The change itself would be a oneliner [here](https://github.com/mongodb/genny/blob/c058d4243dcb4ce181eaaa422fafd8c99df606cb/src/cast_core/src/RunCommand.cpp#L92).

Although it could also cause a change in performance on some tasks.

Reply:
I opened up a ticket around this a couple of days ago and Ryan seems to agree with the change:
https://jira.mongodb.org/browse/EVG-18233

Reply:
Would it be worth doing  EVG-18233 first then? This PR would be much smaller as a result.